snorkel/end_model/batch_schedulers/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .sequential_scheduler import SequentialScheduler
+from .shuffled_scheduler import ShuffledScheduler
+
+batch_schedulers = {"sequential": SequentialScheduler, "shuffled": ShuffledScheduler}
```

Comment:
Can we define this in trainer.py instead?

Reply:
The point of moving it here was to get it out of trainer.py. If it's in trainer.py, then trainer.py needs to import all scheduler types and whenever a new scheduler is added, trainer.py needs to change.

Reply:
k fine